SPEC.md

```diff
@@ -144,30 +144,30 @@ The key words "must", "must not", "required", "shall", "shall not", "should", "s
 
 #### CDI Devices
 
-The `cdiDevices` field describes the set of hardware devices that can be requested by the container runtime user.
+The `devices` field describes the set of hardware devices that can be requested by the container runtime user.
 Note: For a CDI file to be valid, at least one entry must be specified in this array.
 
-  * `cdiDevices` (array of objects, REQUIRED) list of devices provided by the vendor.
+  * `devices` (array of objects, REQUIRED) list of devices provided by the vendor.
     * `name` (string, REQUIRED), name of the device, can be used to refer to it when requesting a device.
       * Beginning and ending with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (\_), dots (.), and alphanumerics between.
       * e.g: `docker/podman run --device foo ...`
     * `nameShort` (array of strings, OPTIONAL), alternative names for the device. Can be used to reduce the CLI verbosity
       * Entries in the array MUST use the same schema as the entry for the `name` field
-    * `containerSpec` (object, OPTIONAL) this field is described in the next section.
+    * `ociEdits` (object, OPTIONAL) this field is described in the next section.
       * This field should only be merged in the OCI spec if the device has been requested by the container runtime user.
 
 
-#### Container Spec
+#### OCI Edits
 
-The `containerSpec` field describes edits to be made to the OCI specification. Currently only three kinds of edits can be made to the OCI specification: `devices`, `mounts` and `hooks`.
+The `ociEdits` field describes edits to be made to the OCI specification. Currently only three kinds of edits can be made to the OCI specification: `devices`, `mounts` and `hooks`.
```

Comment:
Continuing from my comments above:
> The `ociEdits` field describes edits to be made to the OCI specification. Currently only three kinds of edits can be made to the OCI specification: `devices`, `mounts` and `hooks`.

would in turn be reworded to (or similar...):
The `containerSpec` field describes device nodes, mounts that need to be made available to containers as well as hooks that need to be executed at a particular time during the container creations stages in order to make a device available to the container.
when an OCI compliant container runtime engine is used this can be achieved by editing the OCI specification.

Reply:
I've been thinking about this too, for now we're looking at OCI edits, but it's definitely something we want to be different!
`ContainerSpec` is probably incorrect too, maybe `ContainerEdits` ?

Reply:
I see `containerSpec` as a generalization, where we specify what needs to be done in order to make a device available in a container. The container spec in this context would refer to CDI (limited to CDI) and not the "grand" container spec as represented by OCI.

If this sub-element (formally known as `containerSpec`) targets specifically OCI edits then its OK to keep it as is IMO.
We would probably be fine with it as long as OCI is the input a container runtime provides the container engine and edits to this spec is the only thing required.

Reply:
+1 for keeping containerSpec name

Reply:
To me the spec really describe "what you should do" (you being the runtime) to make this device available to the container. Optionally, if you can't instruct the runtime with mounts, environment variables, ... you can specify hooks to do "what is needed" to make this device available to the container